### PR TITLE
chore: add volunteer hady68

### PR DESCRIPTION
**Pull request by hady68**

### CODE REVIEW

File: data/users/hady68.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Hardikk Kamboj",
+  "bio": "Community Manager | Web Dev ",
```
Comment (codewithvoid) [inline data/users/hady68.json:3]: Choose one title here..Web Dev as platform is more developers focused :)

File: data/users/hady68.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "image": "https://avatars.githubusercontent.com/u/64458111?v=4",
+  "categories": [
+    {
+      "slug": "programming | software-development | web-development | Community-management",
+      "topics": ["JavaScript", "Community management", "React"]
```
Comment (codewithvoid) [inline data/users/hady68.json:8]: technical topics here too

Also, you need to choose one category in slug. E.g for you it should be like

```json
[
  {
    "slug": "web-development",
    "topics": [
      "ReactJS"
    ]
  },
  {
    "slug": "programming",
    "topics": [
      "JavaScript"
    ]
  }
]
```

File: data/users/hady68.json (hunk 1 above)
Comment (hady68) [reply data/users/hady68.json:3]: Alright will change !

File: data/users/hady68.json (hunk 2 above)
Comment (hady68) [reply data/users/hady68.json]: Have made the suggested changes

File: data/users/hady68.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Hardikk Kamboj",
+  "bio": "Web Dev ",
```
Comment (codewithvoid) [inline data/users/hady68.json:3]: remove extra space in suffix

Comment (hady68) [reply data/users/hady68.json:2]: Made the changes.

Comment (codewithvoid) [reply data/users/hady68.json:2]: You didn't push ig. Check once.

**Comment by hady68** [comment]
@codewithvoid  have committed please check

**Comment by codewithvoid** [comment]
Thanks for volunteering @hady68

**Pull request merged**